sql/core/src/main/scala/org/apache/spark/sql/RuntimeConfig.scala

```diff
@@ -86,6 +86,10 @@ class RuntimeConfig private[sql](sqlConf: SQLConf = new SQLConf) {
     sqlConf.getConf(entry)
   }
 
+  protected[sql] def getOption[T](entry: OptionalConfigEntry[T]): Option[T] = {
```

Comment:
I'd prefer to keep things similar to `SparkConf` and have this method be just called `get`.